README.md

```diff
@@ -31,7 +31,6 @@ Here's an example initialization of TinyMCE. Note that you need to load equation
 Anywhere that you display text that could include equations, you will need to include the Mathquill JS and CSS.
 
 ```javascript
-tinymce.PluginManager.load('equationeditor', '/path/to/plugin.min.js');
```

Comment:
Does the equation editor JS need to be at a specific path for it to be automatically loaded now?

Reply:
tinymce will detect plugin.js or plugin.min.js if `equationeditor` was pointed in tinymce.